include/reactor-uc/action.h

```diff
@@ -13,52 +13,20 @@ struct Action {
   Trigger super;          // Inherit from Trigger
   interval_t min_offset;  // The minimum offset from the current time that an event can be scheduled on this action.
   interval_t min_spacing; // The minimum spacing between two consecutive events on this action.
-  tag_t previous_event;   // Used to enforce min_spacing
-  TriggerEffects effects; // The reactions triggered by this Action.
-  TriggerSources sources; // The reactions that can write to this Action.
-  TriggerDataQueue trigger_data_queue; // FIFO storage of the data associated with the events scheduled on this action.
+  bool is_physical;
+  tag_t previous_event; // Used to enforce min_spacing
```

Comment:
`min_spacing` is part of the reactor semantics. It can be useful because you bound the "event bandwidth" through a port. If `min_spacing` is 100msec then there will maximum be 10 events/sec scheduled on this action. 

But it is not correctly implemented now since we support arbitrary scheduling. We would need to search the event queue and look for events from the same action happening before and after...

I think we can safely remove it for now. Thanks for comment

Reply:
Note that validating real-time properties is not possible (for physical actions at least) without the `min_spacing` parameter.

Reply:
Yes, I agree. It is useful, I will put it on the backlog to implement later.